[docs][explore] Update SSL steps in build an app section for Java (#10836)
yugabyte/yugabyte-db: docs/content/latest/quick-start/build-apps/java/ysql-jdbc-ssl.md

- Java Development Kit (JDK) 1.8 or later is installed. JDK installers for Linux and macOS can be downloaded from [OpenJDK](http://jdk.java.net/), [AdoptOpenJDK](https://adoptopenjdk.net/), or [Azul Systems](https://www.azul.com/downloads/zulu-community/).
- [Apache Maven](https://maven.apache.org/index.html) 3.3 or later is installed.
- [OpenSSL](https://www.openssl.org/) 1.1.1 or later is installed.

## SSL certificates for a cluster in Kubernetes (Optional)

1. Create a minikube cluster by adding `tls.enabled=true` to the command line described in [Quick start](../../../../quick-start/create-local-cluster/kubernetes/).

   ```sh
   $ kubectl create namespace yb-demo
   $ helm install yb-demo yugabytedb/yugabyte \
   --set resource.master.requests.cpu=0.5,resource.master.requests.memory=0.5Gi,\
   resource.tserver.requests.cpu=0.5,resource.tserver.requests.memory=0.5Gi,\
   replicas.master=1,replicas.tserver=1,tls.enabled=true --namespace yb-demo
   ```

1. Verify that SSL is enabled using `ysqlsh`.

>    ```sh
>    $ ysqlsh
>    ysqlsh (11.2-YB-2.9.0.0-b0)
>    SSL connection (protocol: TLSv1.3, cipher: TLS_AES_256_GCM_SHA384, bits: 256, compression: off)
>    Type "help" for help.
>    ```

Reviewer: ````suggestion
    ```sh
    $ ysqlsh
    ```

    ```output
    ysqlsh (11.2-YB-2.9.0.0-b0)
    SSL connection (protocol: TLSv1.3, cipher: TLS_AES_256_GCM_SHA384, bits: 256, compression: off)
    Type "help" for help.
    ```
````

    $ keytool -keystore ybtruststore -alias ybtruststore -import -file ca.crt
    ```

    Enter a password when you're prompted to enter one for your keystore.

> 1. Export the truststore, `<YOURSTOREPASS>` should be the password you would have used for your keystore creation.

Reviewer: ```suggestion
1. Export the truststore. In the following command, replace `<YOURSTOREPASS>` with the password you used for your keystore creation.
```
